src/openfermion/utils/_remove_symmetry_qubits_test.py

```diff
@@ -0,0 +1,171 @@
+""" Unit tests of remove_symmetry_qubits().
+    Test that it preserves the correct
+    ground state energy, and reduces the number
+    of qubits required by 2.
+"""
+
+import unittest
+
+import openfermion
+import openfermionpyscf
+from openfermion.hamiltonians import MolecularData
+from openfermionpyscf import run_pyscf
+from openfermion.utils import eigenspectrum
+from openfermion.ops import FermionOperator, QubitOperator
+from openfermion.transforms import get_fermion_operator
+
+import _remove_symmetry_qubits
```

Comment:
you should edit the __init__ file in utils to import your function and then import it through utils.